packages/ui/components/head-seo/head-seo.test.tsx

```diff
@@ -45,9 +45,6 @@ vi.mock("@calcom/lib/OgImages", async () => {
     constructGenericImage() {
       return "constructGenericImage";
     },
-    constructMeetingImage() {
```

Comment:
- removing this because meeting prop is removed from `HeadSeo`.
- `HeadSeo` will be completely removed once we completely migrate to App Router